src/main/resources/standardTestCases/empty_string/empty_string.wdl

```diff
@@ -0,0 +1,33 @@
+task hello {
+  command {
+    echo "Hello!"
+  }
+  output {
+    String empty = ""
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+}
+
+task goodbye {
+  String emptyInputString
+  command {
+    echo "${emptyInputString}"
+  }
+  output {
+    String empty = read_string(stdout())
+  }
+  runtime {
+   docker: "ubuntu:latest"
+  }
+}
+
+workflow hello {
+  call hello
+  call goodbye {input: emptyInputString=hello.empty }
```

Comment:
Might as well throw in empty literals too:
`call goodbye {input: emptyInputString=""}`
And empty workflow inputs:
`call goodbye {input: emptyInputString=emptyWfInput}`